.github/workflows/release_linux.yml

```diff
@@ -0,0 +1,41 @@
+name: Draft Release Linux
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+env:
+  GITHUB_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
+  GH_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
+  CSC_IDENTITY_AUTO_DISCOVERY: true
```

Comment:
Those should not be needed, they are only for macOS signing.
We just need the github ones.